src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -60,6 +66,11 @@ public class RoleManager<TRole> : IDisposable where TRole : class
         protected IRoleStore<TRole> Store { get; private set; }
 
         /// <summary>
+        ///     Used to create scopes and messages for logging
+        /// </summary>
+        protected internal virtual IdentityLogger IdentityLogger { get; set; }
```

Comment:
Does anyone actually call Logger.Logger?  The point is that the manager usage looks worse with it called IdentityLogger, that should be the usage pattern you are optimizing for, as its IdentityLogger all over the place right now instead of Logger which is much better